src/vmspawn/vmspawn-util.c

```diff
@@ -33,6 +33,19 @@ OvmfConfig* ovmf_config_free(OvmfConfig *config) {
         return mfree(config);
 }
 
+int parse_qemu_network_stack(const char *s, QemuNetworkStack *net_stack) {
+        if (streq(optarg, "user"))
```

Comment:
assert(s)
assert(net_stack)

Reply:
also did you mean s here instead of optarg?

Reply:
Fixed in ffc1867558.

Reply:
Still not fixed - please use `s` instead of `optarg`.